src/components/MovieList/MovieList.jsx

```diff
@@ -1,5 +1,13 @@
+import { MovieCard } from '../MovieCard';
 import './MovieList.scss';
 
-export const MovieList = () => (
-  <>Put the list here</>
+export const MovieList = ({ movies }) => (
+  <div className="movies">
+    {movies.map(movie => (
```

Comment:
There is no check for if the movies prop is null or undefined. It would be a good idea to add a check to prevent errors if the movies array is not passed in as a prop or is undefined.